src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -39,11 +39,6 @@ export default class CodeSnippet extends Component {
      */
     feedback: PropTypes.string,
 
-    /**
-     * Specify the label used for the Copy Button
-     */
-    copyLabel: PropTypes.string,
```

Comment:
The removal here seems a breaking change - Could we put a compatibility code to support `copyLabel` until next major?

Reply:
Hmm, do you mean if they set the `copyLabel` prop we send a warning saying "This prop is non-functional and will be deprecated in Carbon 11 👋🏽"?

Reply:
I see, if your intent is creating a `v11` code then we can add `breakingChangesXI` feature flag to `src/internal/FeatureFlags.js` and we can run new code _only_ when the flag is true.

Another way to deal with this kind of thing is making `copyLabel` work (in addition to `ariaLabel`) until we ship `v11`.

Reply:
Got it. Thanks for clarifying 👍

Reply:
Thanks @dakahn for the update! My last comment to the latest code is that we don't want to remove `ariaLabel`, either (for the sake of backward-compatibility) - So I'm anticipating something like this:

```diff
--- CodeSnippet.js.orig	2019-04-10 23:31:26.000000000 -0500
+++ CodeSnippet.js	2019-04-10 23:37:02.000000000 -0500
@@ -55,6 +55,12 @@
      * Specify a label to be read by screen readers on the containing <textbox>
      * node
      */
+    ariaLabel: PropTypes.string,
+
+    /**
+     * Specify a label to be read by screen readers on the containing <textbox>
+     * node
+     */
     copyLabel: PropTypes.string,
 
     /**
@@ -114,7 +120,8 @@
       children,
       feedback,
       onClick,
-      copyLabel,
+      ariaLabel,
+      copyLabel, // TODO: Merge this prop to `ariaLabel` in `v11`
       copyButtonDescription,
       light,
       showMoreText,
@@ -170,7 +177,7 @@
         role="textbox"
         tabIndex={0}
         className={`${prefix}--snippet-container`}
-        aria-label={copyLabel ? copyLabel : 'code-snippet'}>
+        aria-label={ariaLabel || copyLabel || 'code-snippet'}>
         <code>
           <pre
             ref={codeContent => {
@@ -195,7 +202,7 @@
         <Copy
           {...other}
           onClick={onClick}
-          aria-label={copyLabel}
+          aria-label={copyLabel || ariaLabel}
           aria-describedby={this.uid}
           className={codeSnippetClasses}
           feedback={feedback}>
```